Server/src/main/java/local/ModelDAOImpl.java

```diff
@@ -72,20 +72,20 @@ public String getModelNameFromAssetTypeID(String assetTypeID) {
      * Given an arraylist of trained models this function will write the new Classifier object
      * to the corresponding trained model entry after an model training phase
      *
-     * @param tms<TrainedModel> represents a list of trained models
+     * @param tm<TrainedModel> represents a list of trained models
      * @author Paul
      */
     @Override
-    public void setModelsToTrain(ArrayList<TrainedModel> tms) {
+    public void setModelsToTrain(TrainedModel tm) {
```

Comment:
It would be good to update the method description and param description to reflect the new changes. Also, the name "setModelsToTrain" could be updated to "setModelToTrain" (singular) to add clarity to the function.